src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
I also seem to remember that. But as Xiao noted, if you can point to gate-per-second numbers on some benchmarks and tell me that it doesn't matter over LAN (or EC2), I'm okay with it.